streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceivedBlockTracker.scala

```diff
@@ -195,30 +219,35 @@ private[streaming] class ReceivedBlockTracker(
       timeToAllocatedBlocks --= batchTimes
     }
 
+    def resolveEvent(event: ReceivedBlockTrackerLogEvent): Unit = {
```

Comment:
This does not need to change. The RBT should not even be aware that there is batching going on in the WAL that is being used.